### PR TITLE
Add News, Release Note, Download Link for Apache Spark 3.5.0

### PR DESCRIPTION
**Pull request by xuanyuanking**
- Vote Result: https://lists.apache.org/thread/kqqlsy90ormkccddtx6wgqmy8r1df8tx
- Documentation: https://spark.apache.org/docs/3.5.0/
- Binary: https://downloads.apache.org/spark/spark-3.5.0/
- PySpark: pending on the File Limit Request https://github.com/pypi/support/issues/3175
- Maven Central: https://repo1.maven.org/maven2/org/apache/spark/spark-core_2.13/3.5.0/

image
image
image

### CODE REVIEW

**Comment by xuanyuanking** [comment]
cc @gengliangwang

**Comment by xuanyuanking** [comment]
cc @Yikun, could you also help me with DockerHub image parts? I don't have the access

**Comment by Yikun** [comment]
https://github.com/apache/spark-docker/pull/55

I already submited the draft patch, need your help to upload the key.

**Review by HyukjinKwon — APPROVED** [review]

**Review by gengliangwang — APPROVED** [review]

File: releases/_posts/2023-09-13-spark-release-3-5-0.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+* Upgrade dev.ludovic.netlib to 3.0.3 [SPARK-41750](https://issues.apache.org/jira/browse/SPARK-41750)
+* Upgrade hive-storage-api to 2.8.1 [SPARK-41798](https://issues.apache.org/jira/browse/SPARK-41798)
+* Upgrade Apache httpcore to 4.4.16 [SPARK-41802](https://issues.apache.org/jira/browse/SPARK-41802)
+* Upgrade jetty to 9.4.52.v20230823 [SPARK-45052](https://issues.apache.org/jira/browse/SPARK-45052)
+* Upgrade compress-lzf to 1.1.2 [SPARK-42274](https://issues.apache.org/jira/browse/SPARK-42274)
+
+### Removals, Behavior Changes and Deprecations
+
+#### Upcoming Removal
+For the The following features will be removed in the next Spark major release
```
Comment (yaooqinn) [inline releases/_posts/2023-09-13-spark-release-3-5-0.md]:
```suggestion
For the the following features will be removed in the next Spark major release
```

**Review by cloud-fan — APPROVED** [review]

**Comment by xuanyuanking** [comment]
Sorry for missing the ping @Yikun ! Just uploaded my key. Thank you!

**Review by yaooqinn — APPROVED** [review]

**Comment by xuanyuanking** [comment]
Thank you all for the review! 🥂

**Pull request closed without merging**